tools/RAiDER/models/ecmwf.py

```diff
@@ -101,11 +101,10 @@ def _load_model_level(self, fname):
         self._q = q
         geo_hgt, pres, hgt = self._calculategeoh(z, lnsp)
 
-        # re-assign lons, lats to match heights
-        _lons = np.broadcast_to(lons[np.newaxis, np.newaxis, :], hgt.shape)
-        _lats = np.broadcast_to(lats[np.newaxis, :, np.newaxis], hgt.shape)
+        self._lons, self._lats = np.meshgrid(lons, lats)
```

Comment:
this is needed for consistency with HRRR; a 2D array should be used

Reply:
You don't necessarily have to use 2D. If memory is limited or you want to keep footprint down, use numpy stride tricks:

https://towardsdatascience.com/advanced-numpy-master-stride-tricks-with-25-illustrated-exercises-923a9393ab20